.github/workflows/check.yaml

```
@@ -4,7 +4,7 @@ jobs:
   check:
     strategy:
       matrix:
-        go-version: [1.19.x]
+        go-version: [1.20.x]
```

Once this workflow starts using `actions/setup-go@v4` instead of `v3` it could be even simplified:
```suggestion
        go-version: ['stable']
```
...but that could have undesired consequences, breaking the checks once a new `stable` go version is released.